### PR TITLE
Misc Multiplayer Sample CFG Cleanup

### PR DESCRIPTION
**Pull request by AMZN-Gene**
Misc CFG cleanup to help prevent workflow-6 (multiplayer) confusion. 
1) Removed unused autoexec.cfg; we can bring this back as needed for auto-launching a starter level for releases, but it wasn't being used and still using old cry map commands. 
2) Added Window's cmd files to partner up with the server/client cfg so users know those cfg's aren't automatically loaded and instead can/need to be used in a command-line.

Partially resolves https://github.com/o3de/o3de/issues/9630

**Testing**
Tested by opening MultiplayerSample editor and making ensuring all the cvars were as expected

Signed-off-by: Gene Walters <genewalt@amazon.com>

### CODE REVIEW

**Comment by AMZN-puvvadar** [comment]
Any concerns around relying on profile builds existing specifically?

**Comment by AMZN-Gene** [comment]
> Any concerns around relying on profile builds existing specifically?

I’m okay with relying on profile builds. I use cmd’s like this myself during development and it’s an example for devs to quickly see how to use cfg’s. Alternatively, we could build out a more complicated/robust script that first searches for release builds, but I think the simplicity is an advantage for educational purposes (and it’ll be what ppl use most the time).

File: launch_client.cmd [hunk 1]
```diff
@@ -0,0 +1 @@
+.\build\windows\bin\profile\MultiplayerSample.GameLauncher.exe --console-command-file=launch_client.cfg
```
Comment (AMZN-Olex) [inline launch_client.cmd:1]: Do we need to update VS debug settings as well?

**Review by allisaurus — COMMENTED** [review]
Love the idea of adding helper cmd scripts, just don't forget to [update the README](https://github.com/o3de/o3de-multiplayersample#running-the-server) with 1) the new .cfg file names, or 2) example usages

File: launch_client.cmd (hunk 1 above)
Comment (AMZN-Gene) [reply launch_client.cmd:12]: good catch, fixed!

**Comment by AMZN-Gene** [comment]
> Love the idea of adding helper cmd scripts, just don't forget to [update the README](https://github.com/o3de/o3de-multiplayersample#running-the-server) with 1) the new .cfg file names, or 2) example usages

Fixed!

File: README.md [hunk 2]
```diff
@@ -83,28 +83,28 @@ This option will output all the project and engine binaries in the engine's buil
...
-
 #### Running the Server
 
 A server can be run as follows
 
 ```shell
-MultiplayerSample.ServerLauncher.exe --console-command-file=server.cfg 
+MultiplayerSample.ServerLauncher.exe --console-command-file=launch_server.cfg 
 ```
+Notice the launch_server.cfg is passed into the commandline. Any file passed into the console-command-file argument will used when starting up the application.
```
Comment (allisaurus) [inline README.md:106]: nit: "will _be_ used..."

**Review by allisaurus — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (AMZN-Gene) [reply README.md]: thank you! fixed!

**Review by AMZN-Olex — APPROVED** [review]

**Pull request merged**